signing_service/signing_service/signing_service.py

```diff
@@ -186,13 +186,15 @@ def _parse_arguments() -> argparse.Namespace:
         help="Concent's public key.",
     )
     parser.add_argument(
+        '-p',
```

Comment:
This is not really related to the protocol. It should be in a separate commit. Or even a separate pull request because this could go to master immediately, without further review.

Reply:
:+1: done